test/e2e/mapml-viewer/viewerContextMenu.test.js

```diff
@@ -226,57 +237,89 @@ test.describe("Playwright mapml-viewer Context Menu (and api) Tests", () => {
     });
   });
 
-  test("Submenu, copy all coordinate systems using tab + enter to access", async () => {
-    await page.click("body > mapml-viewer");
-    await page.keyboard.press("Shift+F10");
-    for (let i = 0; i < 3; i++)
-      await page.keyboard.press("Tab");
-
-    await page.keyboard.press("Enter");
-
-    await page.click("#mapml-copy-submenu > button:nth-child(10)");
-
-    await page.click("body > textarea");
-    await page.keyboard.press("Control+v");
-    const copyValue = await page.$eval(
-      "body > textarea",
-      (text) => text.value
+  test("Submenu, copy using tab + enter to access", async () => {
+    await page.reload();
+      let expected = "";
+    const currDefCS = await page.$eval(
+      "body > mapml-viewer",
+      (map) => ({ext: map._map.contextMenu.defExtCS, loc: map._map.contextMenu.defLocCS})
     );
-    let expected = "z:1\n";
-    expected += "tile: i:30, j:50\n";
-    expected += "tilematrix: column:6, row:6\n";
-    expected += "map: i:250, j:300\n";
-    expected += "tcrs: x:1566, y:1586\n";
-    expected += "pcrs: easting:562957.94, northing:3641449.50\n";
-    expected += "gcrs: lon :-62.729466, lat:80.881921";
-
-    expect(copyValue).toEqual(expected);
-  });
-
-  test("Submenu, copy all coordinate systems", async () => {
-    await page.click("body > mapml-viewer");
-    await page.keyboard.press("Shift+F10");
-    await page.keyboard.press("c");
-
-    await page.click("#mapml-copy-submenu > button:nth-child(10)");
-
-    await page.click("body > textarea");
-    await page.keyboard.press("Control+a");
-    await page.keyboard.press("Backspace");
-    await page.keyboard.press("Control+v");
-    const copyValue = await page.$eval(
-      "body > textarea",
-      (text) => text.value
+    // set cs to pcrs for copying extent test, gcrs for copying location test 
+    for (let i = 0; i < 4; i++) {
```

Comment:
I think the tests would be clearer if the looping was omitted and we replaced the one test with 4 or 5 linear tests that are independently named.